docs/admin/project.md

```diff
@@ -0,0 +1,84 @@
+---
+description: Guide on how to manage projects and quotas as a Hopsworks administrator
+---
+
+# Manage Projects
+
+Hopsworks provides administrator with a view of the projects in a Hopsworks deployment.
+
+Hopsworks administrators are not automatically members of all the projects in a deployment however they can see which projects exist, who is their owners and they can limit the amount of storage and compute quotas each project has available. 
+
+## Prerequisites
+
+You need to be an administrator on a Hopsworks deployment. 
+
+## Changing a project quotas
+
+You can find the Project management page by clicking on your name, in the top right coner of the navigation bar, and choosing _Cluster Settings_ from the dropdown menu and going to the _Project_ tab.
+
+<figure>
+  <img src="../../assets/images/admin/projects/project_list.png" alt="Project page" />
+  <figcaption>Project page</figcaption>
+</figure>
+
+This page will list all the projects in a cluster, their name, owner and when its quota was last updated. By clicking on the _edit configuration_ link of a project you will be able to edit the quotas of that project.
+
+<figure>
+  <img src="../../assets/images/admin/projects/project_quotas.png" alt="Project quotas" />
+  <figcaption>Project quotas</figcaption>
+</figure>
+
+### Storage
+
+Storage quota represents the amount of data a project can store. The storage quota is broken down in three different areas:
+
+- **Feature Store**: This represents the storage quota for files and directories stored in the `_featurestore.db` dataset in the project. This dataset contains all the feature group offline data for the project.
+- **Hive DB**: This represents the storage quota for files and directories stored in the `[projectName].db` dataset in the project. This is a general purpose Hive database for the project that can be used for analytics.
+- **Project**: This represents the storage quota for all the data stored on any ohter dataset.
+
+Each storage quota is divided into space quota, i.e. how much space the files can consume, and namespace quota, i.e. how many files and directories there can be. If Hopsworks is deployed on-premise using hard drives to store the data, i.e. Hopsworks is not configured to laverage a S3-compliant storage system, the data is replicated across multiple nodes (by default 3) and the space quota takes the replication into consideration. As an example, a 100MB file stored with a replication factor of 3, will consume 300MB of space quota.
```

Comment:
i.e., 
always a comma after "i.e.," and "e.g.,"

laverage -> store its data in 
the replication into consideration -> the replication factor into consideration